src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -117,6 +117,16 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
         /// </summary>
         bool IsReadOnly { get; }
 
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
```

Comment:
This same #pragma warning disable exists in the SymbolDisplay helpers directly that these implementations forward to; I'm assuming that if it's OK there, it's OK here too.